content/_code-samples/freeze/py/check_global_freeze.py

```diff
@@ -0,0 +1,38 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountInfo
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # Connect to testnetwork
+
+
+ACCOUNT_ROOT_LEDGER_FLAGS: dict[str, int] = {
+        "lsfNoFreeze": 0x00200000,
+        "lsfGlobalFreeze": 0x00400000,
+    }
+
+def parse_account_root_flags(flags: int) -> list[str]:
+    flags_enabled = []
+    for flag in ACCOUNT_ROOT_LEDGER_FLAGS:
+        check_flag = ACCOUNT_ROOT_LEDGER_FLAGS[flag]
+        if check_flag & flags == check_flag:
+            flags_enabled.append(flag)
+    return flags_enabled
+
+# Issuer address to query for global freeze status
+issuer_addr = "rfDJ98Z8k7ubr6atbZoCqAPdg9MetyBwcg"
+
+# Build account line query
```

Comment:
Similar to previous suggestion - just making it clear in the commandline if you just run the snippet what specifically we are checking. 
```suggestion
# Build account line query
print(f"Checking if global freeze is enabled for the address {issuer_addr}")
```